rts/lra/lra-proxy/api/src/main/java/io/narayana/lra/client/internal/proxy/ProxyService.java

```diff
@@ -249,27 +250,22 @@ public void unregisterDeserializer(LRAParticipantDeserializer deserializer) {
         }
     }
 
-    private static Optional<LRAParticipant> deserializeParticipant(final LRAParticipantDeserializer deserializer, URL lraId, final String objectAsString) {
+    private static Optional<LRAParticipant> deserializeParticipant(URL lraId, final String objectAsString) {
         final byte[] data = Base64.getDecoder().decode(objectAsString);
 
         try (ObjectInputStream ois = new ObjectInputStream(new ByteArrayInputStream(data))) {
             return Optional.of((LRAParticipant) ois.readObject());
         } catch (final IOException | ClassNotFoundException e) {
-            if (deserializer != null) {
-                LRAParticipant participant = deserializer.deserialize(lraId, objectAsString.getBytes());
-
-                if (participant != null)
-                    return Optional.of(participant);
-            }
-
             for (LRAParticipantDeserializer ds : deserializers) {
                 LRAParticipant participant = ds.deserialize(lraId, data);
 
                 if (participant != null)
                     return Optional.of(participant);
             }
 
-            LRAProxyLogger.i18NLogger.error_cannotDeserializeParticipant(deserializer, e);
```

Comment:
It may be used elsewhere but I see you are no longer using error_cannotDeserializeParticipant and didn't remove it from lraI18NLogger - wdyt?

Reply:
Agreed